.gitignore

```diff
@@ -3,7 +3,7 @@
 *.pyc
 *.idea
 generated_ssl_files/
-error_files/
+troubleshooting/
```

Comment:
happy to change this dir name, @arodoni we will need a change here: https://docs.confluent.io/ansible/current/ansible-troubleshooting.html

"Ansible Control Node inside the error_files/ directory"

Reply:
Can go in the 6.1.x branch as well, this change will get released with 6.1.2